Common/src/CConfig.cpp

```diff
@@ -1150,7 +1150,9 @@ void CConfig::SetConfig_Options() {
 
   /*!\par CONFIG_CATEGORY: FluidModel \ingroup Config*/
   /*!\brief FLUID_MODEL \n DESCRIPTION: Fluid model \n OPTIONS: See \link FluidModel_Map \endlink \n DEFAULT: STANDARD_AIR \ingroup Config*/
-  addEnumOption("FLUID_MODEL", Kind_FluidModel, FluidModel_Map, STANDARD_AIR);
+  addEnumOption("FLUID_MODEL", Kind_FluidModel, FluidModel_Map, COOLPROP);
```

Comment:
Please dont change the default option.

Reply:
I recovered the default option, thank you.